src/contract_wrappers/modules/permission_manager/general_permission_manager_wrapper/3.0.0.ts

```diff
@@ -152,6 +155,22 @@ interface ChangePermissionMultiParams extends TxParams {
   valids: boolean[];
 }
 
+/**
```

Comment:
`AddDelegateMultiParams` and `DeleteDelegateMultiParams` shouldn't be here. They are from 3.1.0 version.